### DISCUSSION TITLE
44회차 반성문

### DISCUSSION THREAD

**Discussion by hyun98** [BDD-CLUB/challenge-every-week-posting#443] [6. 반성문]
반성문 미리씁니다
이번 주는 식도염에 몸살 감기까지 같이 와서 죽은 듯이 지냈네요..
다들 스트레스 관리 잘 하시구요, 건강 잘 챙기시길 바랍니다ㅠㅜ

**Comment by shkisme** [comment]
ㅠㅠ 쾌유하시길 바랍니다 !
